runtime/common/src/lib.rs

```diff
@@ -655,6 +656,27 @@ macro_rules! impl_config_traits {
             type WeightInfo = weights::pallet_collective::WeightInfo<Runtime>;
         }
 
+        impl pallet_contracts::Config for Runtime {
+            type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
+            type CallFilter = frame_support::traits::Nothing;
+            type CallStack = [pallet_contracts::Frame::<Runtime>; 5];
```

Comment:
I must admit I don't have any experience writing ink!, but five callstack depth seems very close to the pain threshold. Are there security concerns at work here?